packages/vats/src/proposals/transfer-proposal.js

```diff
@@ -1,6 +1,7 @@
 // @ts-check
 import { E } from '@endo/far';
-import { BridgeId as BRIDGE_ID, VTRANSFER_IBC_EVENT } from '@agoric/internal';
+import { BridgeId as BRIDGE_ID } from '@agoric/internal';
+import { VTRANSFER_IBC_EVENT } from '@agoric/internal/src/action-types.js';
```

Comment:
I am really confused why the transfer vat needs to be aware of the action type in the first place. This seems like a layering violation. No other action type leaks past the cosmic-swingset layer.